src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/TestForEmptyStringsUsingStringLength.Fixer.cs

```diff
@@ -55,18 +56,18 @@ public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
             }
         }
 
-        private FixResolution? TryGetFixResolution(SyntaxNode binaryExpressionSyntax, SemanticModel model)
+        private FixResolution? TryGetFixResolution(SyntaxNode binaryExpressionSyntax, SemanticModel model, CancellationToken cancellationToken)
         {
             bool isEqualsOperator = IsEqualsOperator(binaryExpressionSyntax);
             SyntaxNode leftOperand = GetLeftOperand(binaryExpressionSyntax);
             SyntaxNode rightOperand = GetRightOperand(binaryExpressionSyntax);
 
-            if (ContainsSystemStringEmpty(leftOperand, model))
+            if (ContainsSystemStringEmpty(leftOperand, model) || ContainsEmptyStringLiteral(leftOperand, model, cancellationToken))
```

Comment:
@mavasani, I think the `ContainsSystemStringEmpty` is now unneeded.
The implementation you referred to for the `ContainsEmptyStringLiteral` checks for a constant value of empty string whatever its source (literal, System.String.Empty, or another constant).

We may remove the `ContainsSystemStringEmpty`, then rename `ContainsEmptyStringLiteral` to be more descriptive to `ContainsEmptyString`, right?